src/main/java/org/skife/jdbi/v2/ClasspathStatementLocator.java

```diff
@@ -203,4 +188,11 @@ private static String left(String str, int len)
         }
         return str.substring(0, len);
     }
+
+    private static final SqlScriptParser SQL_SCRIPT_PARSER = new SqlScriptParser(new SqlScriptParser.TokenHandler() {
```

Comment:
Ah, so, should this be a `ThreadLocal` maybe?  [Are Parser instances reentrant?](https://stackoverflow.com/questions/19325803/are-antlr-parsers-for-java-thread-safe)